DEVDOC.md

```diff
@@ -137,7 +137,13 @@ Follow these steps to use the delve debugger for Go with VSCode to debug `gapis`
 
 2. Create a `launch.json` file under the workspace directory with `Ctrl + Shift + P` and `Debug: Open launch.json`
 
-3. Paste the following as one of the launch configurations. This will ensure that there is a launch configuration for attaching to Delve.
+3. There will be two settings file, one is for global the other one is local to the project.
+	- Add this line to your global settings to ensure you have a stable tools directory:
+	`"go.toolsGopath": "<path-to-go-plugin-tools-folder>",`
```

Comment:
Do you have an example of such a plugin tools folder path? (It's the first time I hear of such a thing, is it VS-Code specific?)

Reply:
I am not sure if this is a VSCode specific thing or does it have other versions in other IDEs but it is the place that the plugin downloads tools into. Setting it globally is not mandatory(otherwise it uses $GOPATH) but it helps to be able to change $GOPATH per project.

Reply:
OK, thanks for the clarification. The plugin here is the VSCode go plugin.

I think go.toolsGopath and go.gopath are not in `launch.json`? If so, these new instructions should not be in the middle of the list related to `launch.json`

Reply:
Oh I missed that. Yes, I will move this to somewhere earlier as a pre-step. With including information about setting files.